spec/models/waste_carriers_engine/order_copy_cards_registration_spec.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module WasteCarriersEngine
+  RSpec.describe OrderCopyCardsRegistration, type: :model do
+    subject(:order_copy_cards_registration) { build(:order_copy_cards_registration, :has_required_data) }
+
+    describe "workflow_state" do
+      context "when a OrderCopyCardsRegistration is created" do
+        it "has the state :copy_cards_form" do
+          expect(order_copy_cards_registration).to have_state(:copy_cards_form)
+        end
+      end
+
+      context "transitions" do
```

Comment:
For WCR renewing regs and for all the WEX journeys, we kept the transition specs in separate files (see https://github.com/DEFRA/waste-exemptions-engine/tree/master/spec/models/waste_exemptions_engine) - should we do that here as well for consistency? I appreciate that the journey here is much simpler, but I think it would be helpful so that we always know where to look for workflow specs.

Reply:
WEX had a lot of shared examples for testing transitions as well - may be worth looking if those would be useful here too.

Reply:
will do :)